project-base/storefront/public/locales/sk/common.json

```diff
@@ -89,9 +89,11 @@
     "Customer information": "Informácie o zákazníkovi",
     "Customized E-shop by": "E-shop na mieru od",
     "Czechia": "Česko",
+    "date of order": "",
```

Comment:
Same for Slovak. I know we are missing some strings in general, but would be nice if you added yours.